devkit/sandbox/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @rpch/sandbox
 
+## 0.2.0
+
+### Changes
+
+- Full rework of the Sandbox to work with new 1.10.0 RPCh
```

Comment:
better to remove version number here as it will be outdated immediately

Reply:
How should I mark the version since which it is currently working/tested?

Reply:
Just don't. It will work with the version in the repo. Once e2e tests work again, we can also readd them into github actions.